schema/playground/collisions.schema

```diff
@@ -0,0 +1,11 @@
+package playground;
+
+type Empty 
+{
+}
+
+component Collisions
+{
+	id = 12008;
+	event Empty player_collided;
+}
```

Comment:
Standard nitpick

Reply:
This happens to me all the time too. I'm going to look for an IDE setting...